Confus/Confus/Networking/ClientConnection.cpp

```diff
@@ -53,19 +54,17 @@ namespace Confus
             }
         }
 
-		void ClientConnection::sendMessage(const std::string& a_Message)
+		void ClientConnection::sendMessage(RakNet::BitStream* a_Stream) 
 		{
 			if(m_Connected)
 			{
-				RakNet::BitStream stream;
-				stream.Write(static_cast<RakNet::MessageID>(EPacketType::Message));
-				stream.Write(a_Message.c_str());
-				m_Interface->Send(&stream, PacketPriority::HIGH_PRIORITY,
+				m_Interface->Send(a_Stream, PacketPriority::HIGH_PRIORITY,
 					PacketReliability::RELIABLE_ORDERED, 0, getServerAddress(), false);
 			}
 			else
 			{
-				m_StalledMessages.push(a_Message);
+                //TODO: Make this work with bitstreams. 
```

Comment:
Can you remove this? Or is this not finished? If not, perhaps just remove the else all and the stalled message queue alltogether?